### ISSUE TITLE
[Sourcing-2][Accessibility: ACC-271.1] List item: 2 items provided with focus indicator at same time

### ISSUE THREAD

**Issue by JyotiKumari1507** [SAP/fundamental-ngx#11295]
#### Is this a bug, enhancement, or feature request?
Bug
#### Describe your proposal.
Component having issue: https://sap.github.io/fundamental-ngx/#/platform/select
Select dropdown and any one of the item of select dropdown are provided with focus indicator at the same time.
Only one UI should receive focus at a time and only that element should be provided with focus indicator.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All latest version

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Click select dropdown and move it with keyboard up/down arrow. focus will be there on select dropdown top and selected item both. 

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Yes we have got this bug from Accessibility expert only.

#### Did you check the documentation and the API?
Yes
#### Is there anything else we should know?
Attaching screenshot for reference
<img width="412" alt="Screenshot 2024-01-21 at 11 02 31 PM" src="https://github.com/SAP/fundamental-ngx/assets/157268666/a06f4c40-fd23-4577-94c5-1320fada0605">
